fineract-provider/src/main/java/org/apache/fineract/useradministration/service/AppUserWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -216,6 +217,28 @@ public CommandProcessingResult updateUser(final Long userId, final JsonCommand c
             	clients = this.clientRepositoryWrapper.findAll(clientIds);
             }
 
+            final String passwordParamName = "password";
+            final String currentPasswordParamName = "currentPassword";
+            final String currentPasswordEncodedValue = command.passwordValueOfParameterNamed(currentPasswordParamName, platformPasswordEncoder,
```

Comment:
would it make sense to rather do this only inside the `if (command.hasParameter(passwordParamName))` ?